### DISCUSSION TITLE
🐛 [BUG] g2plot 2.x 的轴标签展示问题（设置的 tickCount 的数为 20，最后几个轴标签缺失）

### DISCUSSION THREAD

**Discussion by rabbit-tian** [antvis/G2Plot#3578] [Q&A]
复现：https://codesandbox.io/s/nifty-hypatia-s7fh4t?file=/index.ts

![image](https://user-images.githubusercontent.com/26690191/212253243-eb7e69f3-19f1-4e51-82be-2f1bf740bd84.png)

* **G2Plot Version**: 2.4.23
* **Platform**: g2plot

**Comment by hustcc** [comment]
符合预期，对于分类的 x 轴，设置 tickcount 本身就没有意义，你这边的需求是什么？我来看看怎么解决？
